buck2/golang/platforms/defs.bzl

```diff
@@ -46,6 +47,7 @@ def _platforms(ctx):
     return [
         DefaultInfo(),
         ExecutionPlatformRegistrationInfo(platforms = [platform]),
+        PlatformInfo(label = str(name), configuration = configuration),
```

Comment:
why the `str(name)` if name should already be a string?

Reply:
Removed the `str(..) call`